### ISSUE TITLE
Upload image API - https://api.cloudinary.com can be misused and taken down instantly

### ISSUE THREAD

**Issue by chiragpandit1** [Bhupesh1x/insta-clone#5]
https://github.com/Bhupesh1x/insta-clone/blob/6100c80c48d8bfa0ae61b5ce5360a9bca089f28e/client/src/Screen/CreatePost.js#L21

Your upload API service is open without any restrictions, https://api.cloudinary.com 
I can easily misuse this service endpoint and upload gigabytes of content of my own and take down your service charging on your credit card. 

Always use such services on server end such that it takes in an argument of bearer token and you know who is the uploader of that image. 

In other words move ```changeImageHandler``` to server with adding bearer token as a parameter such that it does not makes an entry at client's network tab

![image](https://user-images.githubusercontent.com/77569458/136688182-fddf9cc9-0a9d-4cdc-b242-404941306682.png)
